### PR TITLE
fix: skip anchors for ruby

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**

### CODE REVIEW

**Review by dandhlee — CHANGES_REQUESTED** [review]
Could you add a prefix for the title and update the goldens as well? Otherwise LGTM.

File: testdata/goldens/ruby/AUTHENTICATION.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
+﻿<!DOCTYPE html>
```
Comment (dandhlee) [inline testdata/goldens/ruby/AUTHENTICATION.html:1]: Think we're missing the yml file for this on `testdata/ruby/obj/api`

File: testdata/goldens/ruby/LICENSE.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
+﻿<!DOCTYPE html>
```
Comment (dandhlee) [inline testdata/goldens/ruby/LICENSE.html:1]: Don't think this file is needed?

File: testdata/goldens/ruby/index.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
+﻿<!DOCTYPE html>
```
Comment (dandhlee) [inline testdata/goldens/ruby/index.html:1]: I think we're missing `index.yml` file or similar on `testdata/ruby/obj/api`

File: testdata/goldens/ruby/toc.html [hunk 4]
```diff
@@ -0,0 +1,633 @@
+﻿
```
Comment (dandhlee) [inline testdata/goldens/ruby/toc.html:1]: It should generate a `toc.yaml` file not `toc.html`.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Goldens are failing, see comments below.

**Comment by TheRoyalTnetennba** [comment]
Sry there were a few things in the gitignored site folder from another branch that were affecting the output of UPDATE_GOLDENS. fixing now.

**Comment by dandhlee** [comment]
No worries! It's easy to get things mixed up when testing this locally :S

**Review by dandhlee — APPROVED** [review]
Everything looks good now! Thanks for the quick fixes :)

**Pull request merged**